src/test/util_tests.cpp

```diff
@@ -19,6 +19,8 @@
 
 using namespace std;
 
+extern map<string, string> mapArgs;
```

Comment:
If we're going to still have it in the tests, why not keep having it declared in one place (util.h) instead of at least 4 places?